server/api/projects.js

```diff
@@ -948,72 +948,77 @@ const getUserProjects = (req, res) => {
  * @param {Object} req - The Express.js request object.
  * @param {Object} res - The Express.js response object.
  */
-const getUserProjectsAdmin = (req, res) => {
-  Project.aggregate([
-    {
-      $match: {
-        $and: [
-          {
-            $or: constructProjectTeamMemberQuery(req.query.uuid),
-          },
-          {
-            status: {
-              $ne: "completed",
+async function getUserProjectsAdmin(req, res) {
+  try {
+    let userid = req.query.uuid;
+    const centralID = req.query.centralID;
+    if (centralID) {
+      const found = await User.findOne({centralID: req.query.uuid});
```

Comment:
Let's add .orFail() to the end of the findOne() here so we can automatically throw error if no user is found -JT